src/main/java/edu/rosehulman/quota/javasdk/QuotaClient.java

```diff
@@ -125,6 +125,35 @@ IncrementQuotaStatus incrementQuota(Partner partner, Product product, User user,
     }
   }
 
+  /**
+   * @param partner
+   * @param product
+   * @param user
+   * @param quota
+   * @param count
+   * @return true if Quota incremented successfully, false otherwise
+   */
+  IncrementQuotaStatus incrementQuota(Partner partner, Product product, User user, Quota quota, BigInteger count) {
```

Comment:
This method is can't be directly called by a partner. You need to add a way to call it like [this](https://github.com/Quota-Billing/Java-SDK/blob/master/src/main/java/edu/rosehulman/quota/javasdk/Quota.java#L24-L26).